.github/labeler.yml

```diff
@@ -0,0 +1,27 @@
+# labeler "full" schema
+# https://github.com/marketplace/actions/auto-labeler
+
+# enable labeler on issues, prs, or both.
+enable:
+  issues: false
+  prs: true
+# comments object allows you to specify a different message for issues and prs
+
+comments:
+  prs: |
+    I have applied any labels matching special text in your title and description.
+
+    Please review the labels and make any necessary changes.
+
+# Labels is an object where:
+# - keys are labels
+# - values are objects of { include: [ pattern ], exclude: [ pattern ] }
+#    - pattern must be a valid regex, and is applied globally to
+#      title + description of issues and/or prs (see enabled config above)
+#    - 'include' patterns will associate a label if any of these patterns match
+#    - 'exclude' patterns will ignore this label if any of these patterns match
+labels:
+  exclude-from-changelog:
+    include:
+      - 'pre-commit\.ci'
```

Comment:
This regex might create a few false positives (PRs wrongly excluded from changelog) but for that it would require to have  `pre-commit.ci` in the title or PR description and for us to forget to remove the label: I think the odds of that happening are pretty low.